clis/teliod/src/main.rs

```diff
@@ -178,6 +451,7 @@ async fn daemon_event_loop(config: TeliodDaemonConfig) -> Result<(), TeliodError
                 match signal {
                     Some(SIGTERM) => {
                         info!("Received SIGTERM signal, exiting");
+                        fs::remove_file("/var/run/teliod.sock")?;
```

Comment:
It should be closed on any kind of exit, not only on such signal

Reply:
Moved this to the end.